examples/run.rs

```diff
@@ -0,0 +1,87 @@
+use async_trait::async_trait;
+use faktory::{Client, Job, JobRunner, Worker};
+use std::io::{Error as IOError, Result as IOResult};
+use std::sync::Arc;
+use tokio::sync::mpsc;
+use tokio::time;
+
+pub struct JobHandler<T> {
+    chan: Arc<mpsc::Sender<T>>,
+}
+
+impl<T> JobHandler<T> {
+    pub fn new(chan: &Arc<mpsc::Sender<T>>) -> Self {
+        Self {
+            chan: Arc::clone(chan),
+        }
+    }
+}
+
+impl JobHandler<u64> {
+    async fn process_one(&self, job: Job) -> IOResult<()> {
+        time::sleep(time::Duration::from_millis(100)).await;
+        let mut args = job.args().into_iter();
+        let x = args
+            .next()
+            .expect("'x' to be some")
+            .as_u64()
+            .expect("'x' to be an integer");
+        let y = args
+            .next()
+            .expect("'y' to be some")
+            .as_u64()
+            .expect("'y' to be an interger");
+        self.chan.send(x + y).await.expect("no error");
+        Ok(())
+    }
+}
+
+#[async_trait]
+impl JobRunner for JobHandler<u64> {
+    type Error = IOError;
+
+    async fn run(&self, job: Job) -> Result<(), Self::Error> {
+        self.process_one(job).await.unwrap();
+        Ok(())
+    }
+}
+
+#[tokio::main]
+async fn main() {
+    tracing_subscriber::fmt::fmt()
+        .with_max_level(tracing::Level::TRACE)
```

Comment:
Why this?

Reply:
I really wanted to have [traces](https://github.com/jonhoo/faktory-rs/blob/a09d7c92abe2a0477c5193573fba06fba8a4eac1/src/worker/health.rs#L66) appear in the console.